projects/packages/my-jetpack/tests/php/test-products-rest.php

```diff
@@ -167,20 +167,31 @@ public function test_get_products_not_logged() {
 	public function test_get_product() {
 		$product = Products::get_product( 'boost' );
 
-		$request = new WP_REST_Request( 'GET', '/my-jetpack/v1/site/products/boost' );
+		$request = new WP_REST_Request( 'GET', '/my-jetpack/v1/site/products' );
+		$request->set_query_params(
+			array(
+				'products' => 'boost',
+			)
+		);
 
 		$response = $this->server->dispatch( $request );
 		$data     = $response->get_data();
+		print_r( $data );
```

Comment:
I think we have a leftover testing line here 😄

Reply:
Oops, thanks for catching that!